### DISCUSSION TITLE
2FA recovery codes are unnecessarily long

### DISCUSSION THREAD

**Discussion by brunnerh** [npm/feedback#659] [Website] [answered]
In other applications the codes are around 8 to 10 chars long, for NPM they are 64 chars long.
One should not need to get a printer to get them committed to paper in a reasonable time frame and without errors 😑

**Comment by monishcm** [comment, marked as answer]
Thanks for the feedback. We are considering to improve the experience and have this in our backlog. Please look forward for an updated experience soon.
